web/themes/new_weather_theme/tests/components/generic-tabs-tests.js

```diff
@@ -0,0 +1,172 @@
+/* eslint no-unused-expressions: off */
+import { expect } from "chai";
```

Comment:
For writing a11y-aware tests in JS, I'm a pretty big fan of https://testing-library.com/ ... not suggesting to rewrite these tests, but maybe to include it later on?

For example, your first button should have an accessible label of `Tab 1 content`. With chai, you have to write a specific assertion for that, but with testing library, you just grab the button by its accessible label (rather than `querySelector`), so if the label doesn't exist in the accessibility tree, the test fails and you don't have to write any extra code at all.

Reply:
Good idea. I might roll that into the tab consolidation ticket #2152